README.md

```diff
@@ -502,6 +503,16 @@ To add a resource or contribute, please check out our contribution guide [here](
 
 ---
 
+## Svelte
+
+- [Svelte Official documentation](https://svelte.dev/tutorial/basics)
+- [Svelte for beginners](https://coursetro.com/posts/code/176/Learn-Svelte-3.0---Svelte-Tutorial-for-Beginners)
+- [Svelte tutorial by MDN](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/Svelte_getting_started)
```

Comment:
Please replace the title with the title of the page which is Getting started with Svelte